pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/DefaultEnvironmentMerger.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.util;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Default implementation of EnvironmentMerger.
+ *
+ * Just updates maps directly.
+ */
+public class DefaultEnvironmentMerger implements EnvironmentMerger {
+    @Override
+    public Map<String, String> mergeEnvironment(Map<String, String> target, Map<String, String> source) {
+        if (source != null) {
```

Comment:
I think you probably want to make a new map, and insert all of the non-null values into it. This will change the origional I feel will cause problems down the road.